examples/api/test/cupertino/expansion_tile/cupertino_expansion_tile.0_test.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_api_samples/cupertino/expansion_tile/cupertino_expansion_tile.0.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('CupertinoExpansionTile transition modes test', (WidgetTester tester) async {
+    await tester.pumpWidget(const CupertinoExpansionTileApp());
+
+    // Check initial labels
```

Comment:
Small grammar nit:
```suggestion
    // Check initial labels.
```

and below.

Reply:
Thanks for pointing that out — updated the grammar accordingly.